### PR TITLE
Corrections made after Review-1 for Task-1

### PR DESCRIPTION
**Pull request by akashdoppalapudi**
1. Made all the Keywords Lowercase
2. Used "--" for single line commands instead of "/* */"
3. added description of query on top of each query as a comment
4. Mentioned column names instead of using *
5. Replaced concat with concat_ws to deal with NULL values in last_name field
6. Removed user-defined functions
7. corrected average salary to average age in query-28
8. used order by instead of subqueries for youngest, eldest employees
9. Wrote a correct query for the 38th question.
10. used "<>" for not equal condition
11. the Last name changed to "Ray" in where clause in query-46
12. Removed cte's and used single select statements

### CODE REVIEW

**Comment by husunu-technovert** [comment]
Merge request description is not meaningful. You just copy pasted my review comments. 
Update the comments as you fixed the issues. 

Eg:
1. Added comments on top of each query
2. Updated all keywords with lower case

File: assignment-01/queries.sql [hunk 1]
```diff
@@ -1,445 +1,447 @@
...
+-- 24
+-- Show the results by adding 5 to ages and removing 250 from salaries of all employees
+select 
     first_name,
     last_name,
     title,
-    ADD_FIVE(age) AS age,
-    SUBTRACT_250(salary) AS salary
-FROM
+    age + 5 as age,
```
Comment (husunu-technovert) [inline assignment-01/queries.sql]: Its best to wrap expressions in paranthesis

File: assignment-01/queries.sql [hunk 2]
```diff
@@ -1,445 +1,447 @@
...
+-- 35
+-- Select the eldest employee's firstname, lastname and age whose salary is less than 35000
+select 
+    first_name, last_name, age
+from
+    employee
+where
+    salary < 35000
+order by age desc
+limit 1;
```
Comment (husunu-technovert) [inline assignment-01/queries.sql]: `limit 0,1` instead of `limit 1`

**Review by husunu-technovert — COMMENTED** [review]
Your commit messages are good and they are atomic. Follow that only going forwards.

1. Single commit for each issue
2. Each commit should have proper message

File: assignment-01/queries.sql (hunk 1 above)
Comment (akashdoppalapudi) [reply assignment-01/queries.sql]: Wrapped expressions with paranthesis

File: assignment-01/queries.sql (hunk 2 above)
Comment (akashdoppalapudi) [reply assignment-01/queries.sql]: Changed `limit 1` to `limit 0,1`

**Pull request merged**